packages/web/src/pages/search/search.tsx

```diff
@@ -162,7 +334,20 @@ const CustomSearch: React.FC = () => {
           const foundProfiles: Record<string, any> = data["foundProfiles"];
           const resultHtml = buildSearchResults(foundProfiles);
 
+          // Search Intel Bulletins from defined Function
+          let bulletinData = await searchBulletins(searchParam);
```

Comment:
Search bulletins doesn't return promise. So we don't need await. It's a sync function.